src/backend/export/outputVendors/googleSheets/googleSheets.ts

```diff
@@ -48,12 +47,34 @@ const createTransactionsInGoogleSheets: ExportTransactionsFunction = async (
   return spreadsheetAppendResult.data;
 };
 
+async function getSheetName(spreadsheetId, oAuthClient) {
+  const sheetNames = await googleSheets.getSheetNames(spreadsheetId, oAuthClient);
+  if (sheetNames.length === 1) {
+    return sheetNames[0];
+  }
```

Comment:
I don't like this, I think.

You have a very specific unexpected logic here, that you can't guess, I think.

(I'm trying to figure what bother me)

I think the user will prefer to get an error, in case there is no `_budget-tracking`, instead of rewrite existed data. (And I know, maybe you can't re-write existed data because in the next actions it will throw because there is unexpected data in the sheet. But this is coupling).

BTW, why not creating the sheet in case it does not exist? (You have a code to create the sheet, you just need to export it to a separated function)
Or getting the sheet name from the user?

Reply:
I agree this is not optimal.

We can choose to expose the sheet name as a setting if we will see that it is requested.

For now I will just throw an error, thats fine.

Reply:
This was a change I made during development that no longer makes sense. Will revert this change